Stripe/PublicHeaders/STPFile.h

```diff
@@ -48,7 +48,7 @@ typedef NS_ENUM(NSInteger, STPFilePurpose) {
 /**
  * Returns the string value for a purpose.
  */
-+ (NSString *)stringFromPurpose:(STPFilePurpose)purpose;
++ (nullable NSString *)stringFromPurpose:(STPFilePurpose)purpose;
```

Comment:
> If it's already possible for it to be nil and it's just not annotated correctly, I think it's ok to change this without it being a major bump or api change (we should treat it as a bug fix and not an api change).

👍 

> Alternatively, we could just change this to return the empty string or "unknown" or something.

Oh yeah, I dug into this option a little bit. I think returning "unknown" will change our form encoding behavior because the `STPFormEncoder` only encodes non-nil values. Not in the `STPFile` case but definitely in the `STPSourceParams`.

> Or more alternatively, we could just leave it as nonnull since theoretically it can't be if you actually pass in an STPFilePurpose. It's kinda only nullable in ObjC where you can pass in ints (and most ObjC won't care about nullability that much) whereas in Swift you can't really get a wrong value?

You can pass in a `STPFilePurposeUnknown` and get `nil` back.